src/Matroids/ChowRings.jl

```diff
@@ -0,0 +1,95 @@
+export chow_ring, select
+
+function chow_ring(M::Matroid, extended=false)
+	Flats = flats(M)
+	sizeFlats = size(Flats)[1]
+	n = size(M.groundset)[1]
+	if(!is_loopless(M))
+		throw("Matroid has loops")
+	end
+	if(sizeFlats<2)
+		throw("matroid has to few flats")
+	end
+	proper_flats = Flats[2:sizeFlats-1]
+	var_names = [replace(string("x_",S), "["=>"{", "]"=>"}", ", "=>",") for S in proper_flats]# var names?
+	if(extended)
+		var_names = [var_names; [replace(string("h_",S), "["=>"{", "]"=>"}", ", "=>",") for S in [proper_flats;[Flats[sizeFlats]]]]]
+	end
+	s = size(var_names)
+
+	ring, vars = GradedPolynomialRing(QQ, var_names)
+	#ring, vars = PolynomialRing(ZZ, var_names) $which ring do we want here
+	I = linear_relations(ring, proper_flats, vars, M)
+	J = quadratic_relations(ring, proper_flats, vars)
+	Ex = []
+	if(extended)
```

Comment:
This is not mentioned on `https://oscar-system.github.io/Oscar.jl/dev/DeveloperDocumentation/styleguide/#Code-formatting` (but should be, I guess?), but: don't use superfluous parenthesis around `if`-expressions:
```suggestion
	if extended
```